src/client/helpers/blocks.js

```diff
@@ -0,0 +1,69 @@
+import Pizzicato from "pizzicato";
+
+const BLOCKS = [
```

Comment:
can it be moved to constants file or another file is required?

Reply:
fix